tests/Lewee.Tests.Common/Lewee.Tests.Common.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net9.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
```

Comment:
This property group is not needed as these properties are in Directory.Build.props